content/doc/developer/crowdin/crowdin-integration.adoc

```diff
@@ -7,7 +7,7 @@ layout: developersection
 Crowdin is an agile translation tool, which provides a modern web interface to propose and review translations with ease, without the need of any prerequisites you normally need, when working with Jenkins views.
 
 This guide aims to provide guidance for project maintainers, how to prepare and setup their project, in order to use crowdin.jenkins.io.
-This guide is not needed for people, who want to translate plugins only and aren't plugin maintainers. Move on to link:../crowdin/[this document] if you want to learn more about using crowdin as a translator.
+This guide is not needed for people, who want to translate plugins only and aren't plugin maintainers. Move on to link:../[this document] if you want to learn more about using crowdin as a translator.
```

Comment:
```suggestion
This guide is not needed for people, who want to translate plugins only and aren't plugin maintainers.
Refer to link:../[this document] if you want to learn more about using Crowdin as a translator.
```
small suggestions, otherwise this looks good, thanks @mPokornyETM